### PR TITLE
Screen 7-day Incidence (EXPOSUREAPP-4512)

### PR DESCRIPTION
**Pull request by LukasLechnerDev**
Implementation of the 7-day incidence screen.

Missing: Info Button not yet leading to info screen.
Known Issue: Text cut off when setting font size to very large

### CODE REVIEW

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/cards/IncidenceCard.kt [hunk 1]
```diff
@@ -23,6 +27,23 @@ class IncidenceCard(parent: ViewGroup) :
...
-        item.stats as IncidenceStats
+
+        viewBinding.value.iconInfo.setOnClickListener {
+            item.onHelpAction.invoke(item.stats)
+        }
+
+        val stats = item.stats as IncidenceStats
+
+        val primaryFigure =
+            stats.keyFigures.firstOrNull { it.rank == KeyFigureCardOuterClass.KeyFigure.Rank.PRIMARY }
```
Comment (jurajkusnier) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/cards/IncidenceCard.kt:38]: `IncidenceStats` has now getter `sevenDayIncidence` and it's always not null

**Review by chiljamgossow — APPROVED** [review]
LGTM, label and info button are addressed in other PRs

**Review by jurajkusnier — CHANGES_REQUESTED** [review]
Looks good, just please update `IncidenceCard` according to comments.

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/cards/IncidenceCard.kt (hunk 1 above)
Comment (LukasLechnerDev) [reply Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/ui/homecards/cards/IncidenceCard.kt]: That's nice. I updated the PR.

**Review by jurajkusnier — APPROVED** [review]

**Pull request merged**